src/utils.ts

```diff
@@ -70,6 +131,9 @@ export function getRelativeFilePath(file: File): string {
   return path.join(...relative, base);
 }
 
+/**
+ * @param file
+ */
```

Comment:
We seem to need more text here - a description and a return type. I'm not sure why it's also complaining about the `file`.

Reply:
Agreed, waiting on Karol for help with these functions because I don't remember how they work. I'd like to do that in a followup though so we don't hold up this PR.